dartsim/src/SDFFeatures.cc

```diff
@@ -534,7 +534,7 @@ Identity SDFFeatures::ConstructSdfLink(
     const Identity &_modelID,
     const ::sdf::Link &_sdfLink)
 {
-  const auto &modelInfo = *this->ReferenceInterface<ModelInfo>(_modelID);
+  auto &modelInfo = *this->ReferenceInterface<ModelInfo>(_modelID);
```

Comment:
Is there a reason why `modelInfo` is no longer `const`?

Reply:
hmm, I think I was passing `modelInfo` to `AddLink` at one point. Reverted back to `const` in https://github.com/ignitionrobotics/ign-physics/pull/227/commits/0bb7d9e9d787d815f7fd3db15aa2da9a815c40b2.